src/api/status/src/server.js

```diff
@@ -43,5 +58,22 @@ if (process.env.PATH_PREFIX) {
   service.router.use('/', serveStatic(path.join(__dirname, '../public')));
 }
 
+// Get Home page
+service.router.get(process.env.PATH_PREFIX, async (req, res) => {
+  res.render('status', {
+    active_dashboard: true,
+    headers: { title: 'Telescope Dashboard' },
+    telescope: { ...(await getGitHubData('Seneca-CDOT', 'telescope')), title: 'Telescope' },
```

Comment:
Run all these async operations in parallel:

```js
const [ a, b, c, d ] = await Promise.all([ a(), b(), c(), d() ])
```

Reply:
Will change